content/deploy/restore/grafana.md

```diff
@@ -18,10 +17,10 @@ This guide shows you how to restore Grafana in your Rhize environment.
 
 Before you start, ensure you have the following:
 
-{{% param pre_reqs %}} 
+{{% param pre_reqs %}}
 - A [Grafana backup]({{< relref "../backup/grafana" >}})
 
-## Procedure
+## Steps
```

Comment:
During the review, i can see that we are not correctly restoring from our procedure. There are two back ups taken, yet we only refer to the one tar file below. We should separate theses:

---

1. Confirm the cluster and namespace are correct.

  {{% param k8s_cluster_ns %}
  
1. Copy the grafana data tar file into the new Grafana Pod within the `/var/lib/grafana` directory.

```bash
$ kubectl cp ./grafana-data-2023-11-01T16.05.53.tar.gz grafana-64cd6db6f4-8txc2:/var/lib/grafana/
```

1. Untar the file

```bash
 $ tar -xvf grafana-data-2023-11-01T16.05.53.tar.gz --directory /
 ```

1. Copy the grafana configuration file into the new Grafana Pod within `/usr/share/grafana/conf` directory.

```bash
$ kubectl cp ./grafana-conf-2023-11-01T16.05.53.tar.gz grafana-64cd6db6f4-8txc2:/usr/share/grafana/conf
```

1. Untar the file 

```bash
$ tar -xvf grafana-conf-2023-11-01T16.05.53.tar.gz --directory /
```

1. Restart the Grafana Deployment.